webpack/assets/javascripts/react_app/routes/common/WithRenderHandler/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import EmptyPage from '../EmptyPage';
+import LoadingPage from '../LoadingPage';
+
+const WithRenderHandler = (
```

Comment:
nitpick: the convention for HOCs are `withRenderHandler`

Reply:
The name `withRenderHandler` is confusing and I don't feel it tell me about what it does.
What about `withManagedData`?

Reply:
it handles which component to render (loading, empty etc..) its not part of the API hoc